primitives/state-machine/src/lib.rs

```diff
@@ -1579,7 +1579,6 @@ mod tests {
 		let mut seed = [0; 16];
 		for i in 0..50u32 {
 			let mut child_infos = Vec::new();
-			&seed[0..4].copy_from_slice(&i.to_be_bytes()[..]);
```

Comment:
the correct fix is probably https://github.com/paritytech/substrate/pull/9747

Reply:
Will do once it's merged into master.